.changeset/chilled-carrots-shake.md

```diff
@@ -0,0 +1,10 @@
+---
+'modular-scripts': major
+'modular-site': major
+'create-modular-react-app': major
+'eslint-config-modular-app': major
+'modular-views.macro': major
+'tree-view-for-tests': major
```

Comment:
We only need to bump up  modular-scripts and create-modular-react-app

Reply:
I've removed the changeset for now until we decide how to release this - I think releasing all of them at `1.0.0` gives a nice clean line to make changes to them following semver. Otherwise we'll end up with some with `0.x` release still.